src/server/symbols.map

```diff
@@ -1403,3 +1403,25 @@ global:
 local: *;
 };
 
+MIR_SERVER_INTERNAL_2.19 {
+global:
+  extern "C++" {
+    mir::DefaultServerConfiguration::set_the_decoration_manager*;
+    mir::Server::set_the_decoration_manager*;
+    mir::shell::decoration::Decoration::?Decoration*;
+    mir::shell::decoration::Decoration::Decoration*;
+    mir::shell::decoration::DecorationNotifier::?DecorationNotifier*;
+    mir::shell::decoration::DecorationNotifier::DecorationNotifier*;
+    mir::shell::decoration::Manager::?Manager*;
+    mir::shell::decoration::Manager::Manager*;
+    non-virtual?thunk?to?mir::DefaultServerConfiguration::set_the_decoration_manager*;
+    non-virtual?thunk?to?mir::shell::decoration::Decoration::?Decoration*;
+    non-virtual?thunk?to?mir::shell::decoration::Manager::?Manager*;
+    typeinfo?for?mir::shell::decoration::Decoration;
+    typeinfo?for?mir::shell::decoration::DecorationNotifier;
+    typeinfo?for?mir::shell::decoration::Manager;
+    virtual?thunk?to?mir::DefaultServerConfiguration::set_the_decoration_manager*;
+    vtable?for?mir::shell::decoration::Decoration;
+    vtable?for?mir::shell::decoration::Manager;
+  };
+} MIR_SERVER_INTERNAL_2.18;
```

Comment:
Only if ABI is preserved do we keep the old stanza.

The mirserver ABI is being broken. Therefore the soname needs to be bumped and _all_ the symbols go into the `MIR_SERVER_INTERNAL_2.19` stanza.